bpf/lib/nodeport.h

```diff
@@ -378,6 +358,7 @@ int tail_nodeport_nat_ipv6(struct __ctx_buff *ctx)
 {
 	int ifindex = DIRECT_ROUTING_DEV_IFINDEX;
 	int ret, dir = ctx_load_meta(ctx, CB_NAT);
+	union v6addr tmp = IPV6_DIRECT_ROUTING;
```

Comment:
Nit: we could probably rename `tmp` to `dr_ipv6_addr` (or something better)?

Reply:
CI is passing so I'll fix in follow up. I've added to the list.